### PR TITLE
Fix bug with feature scope

### PR DESCRIPTION
**Pull request by Hukumister**

### CODE REVIEW

File: feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/di/WordFlowComponent.kt [hunk 1]
```diff
@@ -59,24 +63,34 @@ interface WordFlowComponent : Injector<WordFlowFragment>,
...
 }
 
+@Module
+interface CreateWordModule {
+    @Binds
+    @PerFlow
+    fun providePhraseTopicRepository(
+        phraseTopicRepositoryImpl: WordRepositoryImpl
+    ): WordRepository
+}
```
Comment (kdk96) [inline feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/di/WordFlowComponent.kt]: should to rename provide method to 'provideWordRepository' and argument too

File: feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/presentation/CreateWordFeature.kt [hunk 2]
```diff
@@ -18,25 +20,26 @@ import ru.coderedwolf.wordlearn.word.model.WordExample
...
+@PerFragment
 class CreateWordFeature @Inject constructor(
-        categoryId: Long,
-        router: FlowRouter,
-        schedulerProvider: SchedulerProvider,
-        wordInteractor: WordRepository
+    categoryId: Long,
+    router: FlowRouter,
+    schedulerProvider: SchedulerProvider,
+    wordInteractor: WordRepository
```
Comment (kdk96) [inline feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/presentation/CreateWordFeature.kt]: May we rename 'wordInteractor' to' wordRepository'?

**Review by kdk96 — CHANGES_REQUESTED** [review]

File: feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/presentation/CreateWordFeature.kt (hunk 2 above)
Comment (Hukumister) [reply feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/presentation/CreateWordFeature.kt]: done

File: feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/di/WordFlowComponent.kt (hunk 1 above)
Comment (Hukumister) [reply feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/di/WordFlowComponent.kt]: done

**Pull request merged**